lib/ContainerManager.js

```diff
@@ -392,15 +408,26 @@ Server.prototype.routes.get['containers'] = function(req, res, next) {
       infos = infos.map(function(info){
         return {
           id: info.Id,
-          name: info.Name,
+          name: info.Name.substr(1), // get rid of initial '/' character
           state: info.State,
           ports: info.NetworkSettings.Ports
         }
       })
 
+      function stateReducer(running){
+        return function(count, info){
+          var state = info.state || info.State
```

Comment:
`state` is the only real code path here. `State` was only there for debugging purposes, I'll remove it.